backend/src/interfaces/requests/auth/login.d.ts

```diff
@@ -12,4 +12,4 @@ type TResponsePayload = ITokens & Pick<IUser, '_id'>;
 type TLoginRequest = TBaseRequest<TRequestParams, TRequestBody>;
 type TLoginResponse = TBaseResponse<TResponsePayload>;
 
-export { TLoginRequest, TLoginResponse };
+export { TLoginRequest, TLoginResponse, TResponsePayload as TLoginPayload };
```

Comment:
If you really need to export this type, pls, leave it's name without `as` alias

Reply:
all those type called TResponsePayload and it could be a little bit confused without specifying

Reply:
File location shows us all information about type purpose (/requests/auth/**login.d.t.s**).

If you have problems with duplicating type names when import, pls, specify `as` alias within imports, not exports:
import { TExample as TLoginExample } from ...